client/actions/client-track.js

```diff
@@ -0,0 +1,48 @@
+'use server'
+
+import axios from'axios';
+
+const getAppointment = async (formEmail, formPhone) => {
+    console.log('running get appointment');
+    try {
+        const data = await axios({
+            url: 'http://localhost:9000/client-track',
+            method: 'GET',
+            headers: {"Content-Type": "application/json"},
+            params: {email: formEmail, phone: formPhone}
+        });
+        console.log('returns', data);
+        console.log('from action', data.data.data);
+        return data.data.data;
+    } catch(err) {
+        console.log(err.response.data);
+        return err.response.data;
```

Comment:
As a consumer, I don't think I'd expect to get an error data if I called `getAppointment()` and it didn't throw. I'd suggest this should either throw in an error state, or we should use a result pattern. Something like this (typescript syntax is just so I can illustrate)
```ts
{
  appointment: Appointment | undefined
  error: unknown | undefined
}
```